### PR TITLE
refactor: Enhance JavaScript parts

### PR DESCRIPTION
**Pull request by flokX**
- Move JavaScript to end
- Replace axios with fetch
- Remove Python 2 benchmarks
- Delete crossdomain.xml because it's not needed

### CODE REVIEW

**Comment by vytas7** [comment]
Thanks for this @flokX ! We'll try to get this merged for the 3.0 release.

**Review by CaselIT — APPROVED** [review]

File: js/dynamics.js [hunk 1]
```diff
@@ -1,19 +1,8 @@
...
-    f();
-  }
-  else {
-    document.addEventListener('DOMContentLoaded', f);
-  }
-}
-
 function setLifetime() {
-  const FALCON_EPOCH = 1357365600;
-  const SECONDS_PER_ORBITAL_ROTATION = 31558149.504000004;
```
Comment (vytas7) [inline js/dynamics.js:12]: I would guess these were supposed to be humorous... Could we maybe restore the original names?

**Review by vytas7 — CHANGES_REQUESTED** [review]
This looks good as far as my limited frontend competence goes... :see_no_evil: 

I would, however, love to restore `FALCON_EPOC` & `SECONDS_PER_ORBITAL_ROTATION`.
These are simply too epic to be renamed. :smiling_imp:

File: js/dynamics.js (hunk 1 above)
Comment (flokX) [reply js/dynamics.js:12]: We can keep that, but I would suggest calculating the _seconds per orbital rotations_ dynamically so it is easier to understand what this variable is doing.

Comment (CaselIT) [reply js/dynamics.js:12]: I think we could keep the previous year length in days of 365,25636.
https://en.wikipedia.org/wiki/Sidereal_year

Comment (flokX) [reply js/dynamics.js:12]: I restored the default value with comment which explains what the variable does.

**Comment by flokX** [comment]
Closing in favour of #31.

**Pull request closed without merging**